core/src/wsv.rs

```diff
@@ -25,7 +25,7 @@ use iroha_logger::prelude::*;
 use iroha_primitives::small::SmallVec;
 
 use crate::{
-    block::Chain,
```

Comment:
we're not using `Chain` anymore? should be removed then

Reply:
I opened this ticket. https://github.com/hyperledger/iroha/issues/2998